conf/db/migration/metrics_portal_ddl/common/V11__create_reporting_tables.sql

```diff
@@ -0,0 +1,73 @@
+-- Copyright 2018 Dropbox, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+CREATE TABLE portal.report_sources (
+    id BIGSERIAL PRIMARY KEY,
+    uuid UUID NOT NULL,
+    version INTEGER NOT NULL DEFAULT 1,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now(),
+    timeout_seconds INTEGER NOT NULL,
+    type VARCHAR(255) NOT NULL,
+
+    -- Chrome
```

Comment:
Is this the actual value of `type` for when these fields apply?

Reply:
No, that was more for my convenience of tracking which field belonged to which instance. I can rename it though to line up with the discriminator value.